src/core_modules/capture-core/components/PossibleDuplicatesDialog/ReviewDialogContents/ReviewDialogContents.container.js

```diff
@@ -0,0 +1,38 @@
+// @flow
```

Comment:
I believe this while file is a copy of the `ReviewDialogContents.container.js` that existed before. I simply moved the folder and github didnt detect the that this is a move of a file.